.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,55 @@
+name: Feature Request
+description: Request any enhancement, or a change to the way Custom Element Plugin CEP works.
```

Comment:
nitpick: just wondering about `enhancement, or` vs `enhancement or`

both correct but got me thinking for a bit

Reply:
I did wonder this myself, I think I will just revert back to not using a comma.